modules/ezcontent_node/modules/ezcontent_smart_article/src/Form/ConfigForm.php

```diff
@@ -64,13 +80,25 @@ class ConfigForm extends ConfigFormBase {
    *   An http client.
    * @param \Drupal\Core\Messenger\Messenger $messenger
    *   A messenger object.
+   * @param \Drupal\ezcontent_smart_article\EzcontentImageCaptioningManager $image_captioning_manager
+   *    A image captioning Manager object.
+   * @param \Drupal\ezcontent_smart_article\EzcontentImageTaggingManager $image_tagging_manager
+   *    A image tagging Manager object.
    */
-  public function __construct(ConfigFactoryInterface $config_factory, EntityTypeManagerInterface $entityTypeManager, FileSystem $fileSystem, ClientInterface $httpClient, Messenger $messenger) {
+  public function __construct(ConfigFactoryInterface $config_factory,
+  EntityTypeManagerInterface $entityTypeManager,
+                              FileSystem $fileSystem,
+  ClientInterface $httpClient,
+  Messenger $messenger,
+                              EzcontentImageCaptioningManager $image_captioning_manager,
+  EzcontentImageTaggingManager $image_tagging_manager) {
     parent::__construct($config_factory);
     $this->fileStorage = $entityTypeManager->getStorage('file');
     $this->fileSystem = $fileSystem;
     $this->httpClient = $httpClient;
     $this->messenger = $messenger;
+    $this->imageCaptioningManager = $image_captioning_manager;
+    $this->imageTaggingManager = $image_tagging_manager;
```

Comment:
The names need to camel cased
```suggestion
    $this->imageCaptioningManager = $imageCaptioningManager;
    $this->imageTaggingManager = $imageTaggingManager;
```